src/applications/financial-status-report/components/otherExpenses/OtherExpensesChecklist.jsx

```diff
@@ -40,19 +39,34 @@ const OtherExpensesChecklist = ({
   };
 
   // Calculate Discretionary income as necessary
-  const updateStreamlinedValues = () => {
-    if (otherExpenses?.length || !gmtData?.isEligibleForStreamlined) return;
```

Comment:
Ahh good catch I missed the removal of that!

Reply:
I don't think we need to prevent it base on length anymore bc we're just running it on page load. `otherExpenses?.length` was to prevent unnecessary calculations when we were handling it on the FE, just running it on mount should be just fine, and we can run it again on the summary page if there's anything added on the checklist